scripts/test-infra/test-e2e-presubmit.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+set -o errexit
+
+# Configure environment
+export IMAGE_REGISTRY="localhost:5001"
+export IMAGE_TAG_NAME=$(git describe --tags --dirty --always)
+export MANIFESTS_DIR="deployment/overlays/default"
+export KUBECONFIG="/tmp/$IMAGE_TAG_NAME"
```

Comment:
Strange filename for kubeconfig 🤔

Reply:
yeah I agree. Maybe something like `export KUBECONFIG="/tmp/$IMAGE_TAG_NAME_kubeconfig"` , WDYT?

Reply:
Yeah, or `export KUBECONFIG="/tmp/kubeconfig_$IMAGE_TAG_NAME` whatever you think is better